From 77ac09ef766e0c042b13e1687a1bcb877dd98dc5 Mon Sep 17 00:00:00 2001
From: Ley Foon Tan <ley.foon.tan@intel.com>
Date: Fri, 14 Feb 2020 02:00:27 +0800
Subject: [PATCH 062/151] ARM: dts: socfpga: arria10: Increase boot partition
 size for NAND

commit  246b1a0b23fc59feaf391403a700459ddf1e40a5 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

[Upstream commit ae0feb88430024938242cc657df48c0657e60bc3]

Increase boot partition size to 32MB to support bigger size kernel image
and FPGA bitstream.

Signed-off-by: Ley Foon Tan <ley.foon.tan@intel.com>
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk_nand.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk_nand.dts b/arch/arm/boot/dts/socfpga_arria10_socdk_nand.dts
index e36e0a0f8aa6..1ebf57004595 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk_nand.dts
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk_nand.dts
@@ -11,10 +11,10 @@
 
 	partition@nand-boot {
 		label = "Boot and fpga data";
-		reg = <0x0 0x1C00000>;
+		reg = <0x0 0x02000000>;
 	};
 	partition@nand-rootfs {
 		label = "Root Filesystem - JFFS2";
-		reg = <0x1C00000 0x6400000>;
+		reg = <0x02000000 0x06000000>;
 	};
 };
-- 
2.26.1

